frontend/13f-app/src/components/results.js

```diff
@@ -17,16 +18,16 @@ function Results () {
   }, [])
   return (
         <div>
-            <table>
+            <table id="companies">
               <tbody>
                 <tr>
                     <th>Company Name</th>
-                    <th>CIK_NO</th>
+                    <th>Number of Filings</th>
                 </tr>
                 {results.map(result => (
                     <tr key={result.cik_no}>
-                        <td>{result.company_name}</td>
-                        <td>{result.cik_no}</td>
+                        <td><Link to={`/company/${result.cik_no}`} id="linkstyle">{result.company_name}</Link></td>
+                        <td id="filingcount">{result.filing_count}</td>
```

Comment:
The [`id`](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Type_Class_and_ID_Selectors#id_selectors) [attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id) and [selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/ID_selectors) are meant for non-repeating elements. Here, you're using an ID selector that will appear multiple times on the results page. Use the [`class`](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Type_Class_and_ID_Selectors#class_selectors) [attribute ](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class) and [selector ](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors) instead.



> An ID selector begins with a # rather than a dot character, but is used in the same way as a class selector. However, an ID can be used only once per page, and elements can only have a single id value applied to them. It can select an element that has the id set on it, and you can precede the ID with a type selector to only target the element if both the element and ID match.

```suggestion
                        <td class="filing-count">{result.filing_count}</td>
```

Also note that I've suggested hyphen-delimiting the words in your selector name. This greatly improves readability (see [this thread](https://stackoverflow.com/a/20811902))

Reply:
Changes made.